packages/nx/src/hasher/hasher.ts

```diff
@@ -283,6 +285,7 @@ class TaskHasher {
   } = {};
   private tsConfigJson: TsconfigJsonConfiguration;
   private nxJson: NxJsonConfiguration;
+  private name: string;
```

Comment:
I don't think this is used? Can we remove it?

Reply:
Good catch. It's leftover from the old take when we deducted scope from package.json's name